mongo_connector/doc_managers/elastic_doc_manager.py

```diff
@@ -137,10 +165,20 @@ def upsert(self, doc, namespace, timestamp):
             "ns": namespace,
             "_ts": timestamp
         }
-        # Index the source document, using lowercase namespace as index name.
-        self.elastic.index(index=index, doc_type=doc_type,
-                           body=self._formatter.format_document(doc), id=doc_id,
-                           refresh=(self.auto_commit_interval == 0))
+
+        parent = self.get_parent(doc_type, doc)
+
+        if parent is None:
+            # Index the source document, using lowercase namespace as index name.
+            self.elastic.index(index=index, doc_type=doc_type,
+                               body=self._formatter.format_document(doc), id=doc_id,
+                               refresh=(self.auto_commit_interval == 0))
+        else:
+            # Index the source document, using lowercase namespace as index name.
+            self.elastic.index(index=index, doc_type=doc_type,
+                               body=self._formatter.format_document(doc), id=doc_id,
+                               refresh=(self.auto_commit_interval == 0), parent=parent)
```

Comment:
I'm *fairly sure* that passing `None` in as the `parent` keyword argument here has the same effect as not passing it at all, so I think you can just use the code in the `else` branch here.